watertap/examples/chemistry/tests/test_pure_water_pH.py

```diff
@@ -654,8 +655,8 @@ def test_solution(self, model_solve):
         pOH = -value(
             log10(model.fs.unit.outlet.mole_frac_comp[0, "OH_-"] * total_molar_density)
         )
-        assert pytest.approx(6.9997414, rel=1e-5) == pH
-        assert pytest.approx(6.9997414, rel=1e-5) == pOH
+        assert pytest.approx(7.0000146, rel=1e-5) == pH
```

Comment:
It looks like our unscaled accuracy here is 1-e3 ? should we not simply relax the rel tolerance?  (For pH 7.001 ~= 6.999) 

I also presume, this change is due to some variables in this model being solved at bound, which now changed slightly as we removed that relaxation, and caused a small propagation into pH.